spec/importer_spec.rb

```diff
@@ -0,0 +1,191 @@
+require 'spec_helper'
+require 'ronin/masscan/importer'
+require 'masscan/output_file'
+require 'ronin/db'
+
+RSpec.describe Ronin::Masscan::Importer do
+  let(:masscan_path) { File.expand_path(File.join(__dir__, '..', 'spec', 'fixtures', 'masscan.json')) }
+  let(:output_file)  { Masscan::OutputFile.new(masscan_path) }
+
+  before(:all) do
+    Ronin::DB.connect('sqlite3::memory:')
+  end
+
+  after(:all) do
+    Ronin::DB::IPAddress.destroy_all
+    # Ronin::DB::Port.destroy_all
+    Ronin::DB::OpenPort.destroy_all
+  end
+
+  describe '.import_file' do
+    let(:expected_values) do
+      [
+        Ronin::DB::IPAddress,
+        Ronin::DB::Port,
+        Ronin::DB::OpenPort,
+        Ronin::DB::IPAddress,
+        Ronin::DB::Port,
+        Ronin::DB::OpenPort,
+        Ronin::DB::IPAddress
+      ]
+    end
+
+    it 'must import records from file' do
+      yielded_values = []
+
+      subject.import_file(masscan_path) do |value|
+        yielded_values << value
+      end
+
+      expect(yielded_values.size).to eq(7)
+      expect(yielded_values.map(&:class)).to match(expected_values)
+    end
```

Comment:
You could probably copy/paste the other `yielded_values` specs from below and reuse them here.